app/models/api/hses.rb

```diff
@@ -16,21 +16,35 @@ def error_type
   def host
     Rails.configuration.x.hses.api_hostname
   end
+
+  def request
```

Comment:
[praise] oh that's awesome that we can move this whole method up into the parent module

Reply:
actually we could move it to the ApiResponse object if we really wanted to (i didn't do it yet cuz i wanted to keep it small) and remove it from TTA

Reply:
(assuming you mean `ApiRequest`) I think that'd be a good thing to keep in our back pocket and do if/when the pattern is the same for monitoring as well.